src/components/Templates/GetNfscCard/GetNfscCard.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Chip } from '../../Core/Chip'
+import { cn } from '../../../utils/cn';
+
+export type GetNfscCardProps = React.ComponentProps<'div'> &
+  Partial<{
+    onClickChip: any,
+    value: number
+  }>;
+
+let cardTitle = "Get $NFSC";
+
+export function GetNfscCard(
+  { onClickChip, value, children, className, ...props }: GetNfscCardProps) {
+    let chips = [20, 50, 100].map(
+      (chipValue) => (<Chip selected={value === chipValue} onSelect={() => onClickChip(chipValue)} >{chipValue}</Chip>));
```

Comment:
when you map, it's always best to use key on the elements

Reply:
Now using chipValue as the key.